astropy/convolution/convolve.py

```diff
@@ -254,8 +254,8 @@ def convolve(array, kernel, boundary='fill', fill_value=0.,
 def convolve_fft(array, kernel, boundary='fill', fill_value=0, crop=True,
                  return_fft=False, fft_pad=None, psf_pad=None,
                  interpolate_nan=False, quiet=False, ignore_edge_zeros=False,
-                 min_wt=0.0, normalize_kernel=False, allow_huge=False,
-                 fftn=np.fft.fftn, ifftn=np.fft.ifftn,
+                 min_wt=0.0, normalize_kernel=False, normalization_rtol=1e-8,
```

Comment:
I know it's just a nitpick but this changes the function signature if the additional parameter isn't added at the end of the list of parameters.

The `mask` was also appended so maybe it wouldn't hurt too much if you place it at the end?